Userland/Libraries/LibJS/Runtime/Intl/NumberFormat.cpp

```diff
@@ -1245,8 +1313,17 @@ ThrowCompletionOr<void> set_number_format_unit_options(GlobalObject& global_obje
 }
 
 // 15.1.14 GetNumberFormatPattern ( numberFormat, x ), https://tc39.es/ecma402/#sec-getnumberformatpattern
-Optional<Variant<StringView, String>> get_number_format_pattern(NumberFormat& number_format, double number, Unicode::NumberFormat& found_pattern)
+Optional<Variant<StringView, String>> get_number_format_pattern(NumberFormat& number_format, Value number, Unicode::NumberFormat& found_pattern)
 {
+    auto as_number = [&]() {
+        if (number.is_number())
+            return number.as_double();
+
+        // FIXME: This should be okay for now as our naive Unicode::select_pattern_with_plurality implementation
+        //        checks against just a few specific small values. But revisit this if precision becomes a concern.
+        return number.as_bigint().big_integer().to_double();
```

Comment:
Should this be a `VERIFY_NOT_REACHED()` in the first commit like all the others?

Reply:
Oops, yeah that was a bad squash